executor/insert_common.go

```diff
@@ -306,18 +322,15 @@ func (e *InsertValues) insertRowsFromSelect(ctx context.Context, exec func(rows
 				return errors.Trace(err)
 			}
 			rows = append(rows, row)
-			if batchInsert && e.rowCount%uint64(batchSize) == 0 {
-				if err := exec(rows); err != nil {
+			if e.rowCount%e.rowCountLimit == 0 {
```

Comment:
I think you should update sessionVar comment for TiDBDMLBatchSize, if you change here.

Reply:
Or use batchSize instead of rowCountLimit

Reply:
The comment of `TiDBDMLBatchSize` doesn't show it is a row size. I think this change won't mislead the comment reader.